test/tests/functional/pbs_reservations.py

```diff
@@ -1603,7 +1604,7 @@ def test_ASAP_resv_request_same_time(self):
         rid1 = self.submit_asap_reservation(user=TEST_USER,
                                             jid=jid2)
         rid1_q = rid1.split('.')[0]
-        exp_attr = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2"),
+        exp_attr = {'reserve_state': (MATCH_RE, "RESV_RUNNING|5"),
```

Comment:
didn't understand this change

Reply:
Actually this is test fix failure i encountered while I was working on these changes, but I will double check whether this change is needed or not...

Reply:
I re-tested this, and with `RESV_CONFIRMED|2`, test some time passes (6/10 times) and some time fails (4/10 times) but with `RESV_RUNNING|5` test always passes. So I think we need this fix.

Reply:
as per offline discussion, I have added proper fix for race condition in test case